internal/pipeline_test.go

```diff
@@ -88,6 +108,8 @@ func (suite *pipelineTestSuite) SetupTest() {
 	AddComponent("ComponentA", NewComponentA)
 	AddComponent("ComponentB", NewComponentB)
 	AddComponent("ComponentC", NewComponentC)
+	AddComponent("stream", NewComponentStream)
```

Comment:
In nightly there is a datavalidation using block-cache state. Let's duplicate that step and run it using stream config to verify things are working fine in case of auto injested block-cache to replace streaming options.